Files/Views/Pages/Properties.xaml.cs

```diff
@@ -78,11 +51,7 @@ private void Properties_Loaded(object sender, RoutedEventArgs e)
                 TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
                 AppSettings.UpdateThemeElements.Execute(null);
             }
-
-            if (BaseProperties != null)
-            {
-                BaseProperties.GetSpecialProperties();
-            }
+            AppSettings.ThemeModeChanged += AppSettings_ThemeModeChanged;
```

Comment:
To fix black window controls (minimize, maximize, close) move `AppSettings.ThemeModeChanged += AppSettings_ThemeModeChanged;` before if in this method.